mhkit/wave/io/cdip.py

```diff
@@ -359,9 +369,15 @@ def request_parse_workflow(
                     silent=silent,
                 )
                 # Cache the individual year's data
-                handle_caching(hash_params, cache_dir, data=year_data)
-            else:
-                year_data = year_data[0]
+                handle_caching(
+                    hash_params,
+                    cache_dir,
+                    cache_content={
+                        "data": year_data,
+                        "metadata": None,
+                        "write_json": None,
+                    },
+                )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args